pkg/spoke/controllers/finalizercontroller/appliedmanifestwork_finalize_controller.go

```diff
@@ -50,7 +51,7 @@ func NewAppliedManifestWorkFinalizeController(
 		WithFilteredEventsInformersQueueKeyFunc(func(obj runtime.Object) string {
 			accessor, _ := meta.Accessor(obj)
 			return accessor.GetName()
-		}, helper.AppliedManifestworkHubHashFilter(hubHash), appliedManifestWorkInformer.Informer()).
+		}, helper.AppliedManifestworkAgentIDFilter(agentID), appliedManifestWorkInformer.Informer()).
```

Comment:
We don't need the `hubHash` filter anymore because the appliedmanifestwork_finalize_controller needs to handle the unmanaged_appliedmanifestwork which is in deleting status, right?

Reply:
it is still needed for appliedmanifestwork_controller.go since it should only handle those with correct hash.